website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Announcing React Native 0.60
+author: Ryan Turner
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://twitter.com/turnrye
+authorImageURL: https://avatars0.githubusercontent.com/u/701035?s=460&v=4
+authorTwitter: turnrye
+category: announcements
+---
+
+After months of hard work from hundreds of contributors, the React Native Core team is proud to announce the release of version 0.60. This release handles significant migrations for both Android and iOS platforms, and many issues are resolved too. This blog post covers the highlights of the release. As always though, refer to the changelog for more detailed information. Finally, thank you contributors for helping us make this milestone!
+
+## Focus on Accessibility
+
+Accessibility APIs have many continued improvements, like `announceForAccessibility`, improvements to roles, better action support, flags, and more. Accessibility is a complex science, but we hope these improvements make it a bit easier to be an A11Y. Be sure to check [React Native Open Source Update June 2019](https://facebook.github.io/react-native/blog/2019/06/12/react-native-open-source-update#meaningful-community-contributions) for more details on these changes.
+
+## A Fresh Start
+
+React Native's start screen has been updated! Thank you to the many contributors who helped create the new UI. This new "Hello World" will welcome users to the ecosystem in a more friendly, engaging way.
+
+![The new init screen helps developers get started from the get-go with resources and a good example](/react-native/blog/assets/0.60-new-init-screen.png)
+
+## AndroidX Support
+
+[AndroidX](https://developer.android.com/jetpack/androidx) is a major step forward in the Android ecosystem, and the old support library artifacts are being deprecated. In 0.60, React Native is migrated over to AndroidX. This is a breaking change, and your native code and dependencies will need to be migrated as well.
+
+> With this change, React Native apps will need to begin using AndroidX themselves. They cannot be used side-by-side in one app, so all of the app code and dependency code needs to be using one or the other.
+>
+> [matt-oakes](https://github.com/matt-oakes) on [discussions-and-proposals](https://github.com/react-native-community/discussions-and-proposals/issues/129)
+
+While your own native code will need to be migrated by you, [@mikehardy](https://github.com/mikehardy), [@cawfree](https://github.com/cawfree), and [@m4tt72](https://github.com/m4tt72) built a [clever tool](https://github.com/mikehardy/jetifier) to patch your `node_modules` while library maintainers are still working on releasing upgrades themselves. So if you find errors related to AndroidX migration, give this a shot.
```

Comment:
"Clever tool" link is not something I want to open, I'd more likely open a link to "jetifier" described as a clever tool. Also please make the statements about migration etc bold, it's something that's gonna break for almost every upgraded app